.github/workflows/validate.yml

```diff
@@ -49,3 +49,45 @@ jobs:
 
       - name: Test
         run: pnpm test
+
+  validate-windows:
```

Comment:
What was the reasoning for splitting this into a separate job rather than doing it in a matrix?

Reply:
Linting works differently on Windows: https://github.com/seek-oss/vocab/actions/runs/4539960941/jobs/8000349114

Reply:
Oh, didn't notice that we weren't linting in this job. 👍